corehq/apps/users/tasks.py

```diff
@@ -341,3 +343,32 @@ def process_reporting_metadata_staging():
     run_again = run_periodic_task_again(process_reporting_metadata_staging_schedule, start, duration)
     if run_again and UserReportingMetadataStaging.objects.exists():
         process_reporting_metadata_staging.delay()
+
+
+@periodic_task(run_every=crontab(minute='*/10'), queue='background_queue')
+def gauge_pending_user_confirmations():
+    metric_name = 'commcare.pending_user_confirmations'
+    from corehq.apps.users.models import Invitation
+    for stats in (Invitation.objects.filter(is_accepted=False).all()
+                  .values('domain').annotate(Count('domain'))):
+        metrics_gauge(
+            metric_name, stats['domain__count'], tags={
+                'domain': stats['domain'],
+                'user_type': 'web',
+            }
+        )
+
+    from corehq.apps.users.analytics import get_inactive_commcare_users_in_domain
+    for doc in Domain.get_all(include_docs=False):
+        domain_name = doc['key']
+        num_unconfirmed = [
+            u.is_account_confirmed
+            for u in get_inactive_commcare_users_in_domain(domain_name)
+        ].count(False)
```

Comment:
Not sure how many there will be, but could avoid allocating a (second) big list
```suggestion
        users = get_inactive_commcare_users_in_domain(domain_name)
        num_unconfirmed = sum(1 for u in users if not u.is_account_confirmed)
```

Reply:
good point. changing this up and testing again in a limited release